layers/+lang/scheme/README.org

```diff
@@ -183,15 +184,41 @@ When enabled the symbol =🆂= should be displayed in the mode-line.
 | ~SPC m e l~ | Evaluate line             |
 | ~SPC m e r~ | Evaluate region           |
 
-** REPL
-
-| Key binding | Description                 |
-|-------------+-----------------------------|
-| ~g j~       | geiser-repl-next-prompt     |
-| ~g k~       | geiser-repl-previous-prompt |
-| ~SPC m h~   | geiser-doc-symbol-at-point  |
-| ~SPC m i~   | geiser-repl-interrupt       |
-| ~SPC m c~   | geiser-repl-clear-buffer    |
-| ~SPC m m~   | geiser-repl-import-module   |
-| ~SPC m u~   | geiser-repl-unload-function |
-| ~SPC m q~   | geiser-repl-exit            |
+** REPL-mode
+
+*Insert state*
+| Key binding | Description           |
+|-------------+-----------------------|
+| ~S-RET~     | Insert newline indent |
+| ~C-l~       | Clear buffer          |
+| ~C-d~       | Exit                  |
+
+*Normal state*
+| Key binding | Description                                       |
+|-------------+---------------------------------------------------|
+| ~g j~       | Goto next prompt                                  |
+| ~g k~       | Goto previous prompt                              |
+| ~]]/[[~     | Goto next/previous prompt                         |
+
+| ~SPC m h d~ | Show documentation for symbol at point            |
+| ~SPC m h c~ | Open customization menu for scheme implementation |
+| ~SPC m C~   | Clear buffer                                      |
+| ~SPC m i l~ | Insert lambda                                     |
+| ~SPC m i m~ | Import module                                     |
+| ~SPC m u~   | Unload function                                   |
+| ~SPC m s~   | toggle surrounding parenthesis <-> brackets       |
```

Comment:
Now I think this is a good binding.
In visual state, `s` binds to `evil-surrond-edit`, which let you change/delete surrounding charater.
Analogously `SPC m s` toggles the surround () to/from [].